src/simulations/ensemblerun.jl

```diff
@@ -2,11 +2,12 @@ export ensemblerun!
 Vector_or_Tuple = Union{AbstractArray,Tuple}
 
 """
-    ensemblerun!(models::Vector, agent_step!, model_step!, n; kwargs...)
+    ensemblerun!(models::Vector, n; kwargs...)
 Perform an ensemble simulation of [`run!`](@ref) for all `model ∈ models`.
 Each `model` should be a (different) instance of an [`AgentBasedModel`](@ref) but probably
 initialized with a different random seed or different initial agent distribution.
-All models obey the same rules `agent_step!, model_step!` and are evolved for `n`.
+All models obey the same rules `agent_step!, model_step!` contained in the model
+and are evolved for `n`.
```

Comment:
In this PR we need to generically shift away from explicitly mentioning `agent_step!, model_step!`. We should say "all models obey the same evolution rules contained in...". In the model documentation we explicitly say "the evolution rules are the functions `agent_step, model_step, scheduler`. If `agent_step == dummy`, the evolution rules is just the function `model_step`.